modules/rostests/apitests/gdi32/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ list(APPEND SOURCE
     CreateBitmapIndirect.c
     CreateCompatibleDC.c
     CreateDIBitmap.c
-    CreateDIBPatternBrush
+    CreateDIBPatternBrush.c
```

Comment:
lol. How does this work?

Reply:
I'd assume cmake authors trying to solve pebkac

Reply:
Maybe ".c" extension is assumed?